OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -25,12 +25,16 @@ public class CombatDebugOverlayInfo : ITraitInfo
 		public object Create(ActorInitializer init) { return new CombatDebugOverlay(init.Self); }
 	}
 
-	public class CombatDebugOverlay : IRenderAboveWorld, INotifyDamage, INotifyCreated
+	public class CombatDebugOverlay : IRenderAboveWorld, INotifyDamage, INotifyCreated, IWorldLoaded, IChatCommand
```

Comment:
Every actor in the world has its own copy of this trait, so I don't expect this will work properly.